components/event-publisher-proxy/testing/utils.go

```diff
@@ -21,10 +21,10 @@ import (
 
 const (
 	// binary cloudevent headers
-	CeIDHeader          = "CE-ID"
-	CeTypeHeader        = "CE-Type"
-	CeSourceHeader      = "CE-Source"
-	CeSpecVersionHeader = "CE-SpecVersion"
+	CeIDHeader          = "Ce-Id"
```

Comment:
Why is this needed as headers are case-insensitive. ?

Reply:
It's the Canonical MIME Header. But in case using `Get()` it's okay to set them to the lower-case ones as defined in the  standard. Here `Get()` changes them on the fly.

Reply:
Done.